website/homepage/upload_demos.py

```diff
@@ -29,21 +31,25 @@ def upload_demo_to_space(
     with tempfile.TemporaryDirectory() as tmpdir:
         demo_path = pathlib.Path(GRADIO_DEMO_DIR, demo_name)
         shutil.copytree(demo_path, tmpdir, dirs_exist_ok=True)
-        if gradio_version:
-            readme = pathlib.Path(tmpdir, "README.md")
-            readme_content = f"""
-                                ---
-                                title: {space_id.split("/")[-1]} 
-                                emoji: 🔥
-                                colorFrom: indigo
-                                colorTo: indigo
-                                sdk: gradio
-                                sdk_version: {gradio_version}
-                                app_file: run.py
-                                pinned: false
-                                ---
-                                """
-            readme.open("w").write(textwrap.dedent(readme_content))
+        readme = pathlib.Path(tmpdir, "README.md")
+        readme_content = f"""
+                            ---
+                            title: {space_id.split("/")[-1]} 
+                            emoji: 🔥
+                            colorFrom: indigo
+                            colorTo: indigo
+                            sdk: gradio
+                            sdk_version: {gradio_version}
+                            app_file: run.py
+                            pinned: false
+                            ---
+                            """
+        readme.open("w").write(textwrap.dedent(readme_content))
+        
+        if gradio_wheel_url:
+            with open(os.path.join(tmpdir, "requirements.txt"), "a+") as r:
+                r.write(gradio_wheel_url)
```

Comment:
I think we need a newline here:

https://huggingface.co/spaces/gradio/fraud_detector_main/blob/main/requirements.txt

Reply:
good catch thanks! fixed: https://huggingface.co/spaces/gradio/fraud_detector_main/blob/main/requirements.txt